### PR TITLE
Make the usage guide for components more clear

### PR DESCRIPTION
**Pull request by pjedrzejewski**

### CODE REVIEW

File: components/general/using_components.rst [hunk 1]
```diff
@@ -14,35 +14,44 @@ Using the Taxation Component
 
 **1.** If you're creating a new project, create a new empty directory for it.
 
-**2.** Open a terminal and use Composer to grab the library.
-
 .. code-block:: bash
 
-    $ composer require sylius/taxation
+    $ mkdir app/
```
Comment (michalmarcinkowski) [inline components/general/using_components.rst]: Maybe `project/`? So nobody confuses it with Symfony `app` dir :)

**Comment by michalmarcinkowski** [comment]
You can also remove the link at the very bottom of this file (`.. _Install composer: https://getcomposer.org/download/`)

File: components/general/using_components.rst (hunk 1 above)
Comment (pjedrzejewski) [inline components/general/using_components.rst]: Good point!

**Comment by pjedrzejewski** [comment]
@michalmarcinkowski Updated.

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**